random_walk.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+np.random.seed(3456)
+# Simulating a random walk 1000 times
+
+all_walks = []
+for x in range(1000):
+    random_walk = [0]
+    for y in range(100):
+        step = random_walk[-1]
+        dice = np.random.randint(1, 7)
+        if dice <= 2:
+            step = max(0, step - 1)
+        elif dice <= 5:
+            step = step + 1
+        else:
+            step = step + np.random.randint(1, 7)
+        if np.random.rand() <= 0.001:
+            step = 0
+        random_walk.append(step)
+    all_walks.append(random_walk)
+
+    np_aw_t = np.transpose(np.array(all_walks))
```

Comment:
I am not familiar with numpy or what transpose does, but `all_walks` is being filled in this cycle, on each iteration you would have all the results and one more, is this expected?